src/app/shared/directives/tooltip.directive.ts

```diff
@@ -17,7 +17,10 @@ export class TooltipDirective implements OnDestroy {
 
   private tooltipComponentRef: ComponentRef<TooltipComponent>
 
-  constructor(private readonly applicationRef: ApplicationRef) {}
+  constructor(
+    private readonly applicationRef: ApplicationRef,
+    private readonly triggerElementRef: ElementRef
```

Comment:
Since this is the `ElementRef` injected into the constructor, we should just call it `elementRef`.

Reply:
Changed to `elementRef`.